packages/create-yoga/create-yoga.ts

```diff
@@ -0,0 +1,210 @@
+import chalk from "chalk";
+import { mkdir } from "fs/promises";
+import path from "path";
+import retry from "async-retry";
+import { isWriteAccess } from "./utils/checkFileAccess.js";
+import {
+  getRepoInfo,
+  hasRepo,
+  existsInRepo,
+  downloadAndExtractRepo,
+  downloadAndExtractExample,
+} from "./utils/example.js";
+import { isFolderEmpty } from "./utils/isFolderEmpty.js";
+import { existsSync, writeFileSync } from "fs";
+import { install } from "./utils/install.js";
+import os from "os";
+
+interface YogaProps {
+  appPath: string;
+  packageManager: any;
+  template?: string;
+  templatePath: string;
+  typescript: string;
```

Comment:
I think we might not need `--typescript` flags, as this is already part of the examples setting?